pyomo/solvers/plugins/solvers/GAMS.py

```diff
@@ -153,36 +153,8 @@ def solve(self, *args, **kwds):
         report_timing=False: bool
             Print timing reports for presolve, solver, postsolve, etc.
         io_options: dict
-            Options that get passed to the writer.
+            Options that get passed to the writer. See writer for details.
```

Comment:
See writer in `pyomo.repn.plugins.gams_writer` for details?

Reply:
Well if looking at the sphinx page, the writer documentation is right below it which is why I put that referring sentence in there. Would the full path the writer be more helpful?

Reply:
I was thinking in the context of someone doing `help(SolverFactory('gams'))`

Reply:
Fair enough